src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/InductionEntity.kt

```diff
@@ -0,0 +1,106 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.induction
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.EntityListeners
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.OneToOne
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotNull
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.UpdateTimestamp
+import org.hibernate.annotations.UuidGenerator
+import org.springframework.data.annotation.CreatedBy
+import org.springframework.data.annotation.LastModifiedBy
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.DisplayNameAuditingEntityListener.CreatedByDisplayName
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.DisplayNameAuditingEntityListener.LastModifiedByDisplayName
+import java.time.Instant
+import java.util.UUID
+
+/**
+ * Represents a Prisoner's Induction, which is typically carried out by a CIAG officer shortly after the Prisoner
+ * has entered a Prison (either when starting a new sentence, or after being transferred from another Prison).
+ */
+@Table(name = "induction")
+@Entity
+@EntityListeners(AuditingEntityListener::class)
```

Comment:
To correctly audit the display names you'll need our entity listener as well as the standard one.
This line should be
```
@EntityListeners(value = [AuditingEntityListener::class, DisplayNameAuditingEntityListener::class])

```